frontend/src/services/http/http.service.ts

```diff
@@ -21,11 +24,16 @@ class Http {
 
   _getHeaders(contentType?: ContentType): Headers {
     const headers = new Headers();
+    const token = localStorage.getItem('token');
```

Comment:
It would be better to use Storage services here

And it would be better to create the `StorageKey` enum instead of just using strings